README.md

```diff
@@ -22,10 +47,35 @@ Paste this on the left side
       }
       subtopics {
         name
-
       }
     }
   }
 }
+```
 
+### Unit tests
+
+Test framework: Jest.
+
+```
+$ yarn test
+```
+
+### Code style
+
+_tl;dr_: Use prettier and eslint!
+
+Format code with prettier to get uniform codestyle:
+
+```
+$ yarn format
 ```
+
+Lint code with [eslint](http://eslint.org/), including [eslint react plugin](https://github.com/yannickcr/eslint-plugin-react), [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import), [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y#readme).
```

Comment:
Kanskje vi kan droppe dette også? Hva med å bare nevne eslint og linke til vår eslint-config?

Reply:
Greit for meg